dnf/cli/commands/__init__.py

```diff
@@ -91,6 +91,9 @@ def _checkEnabledRepo(base, possible_local_files=[]):
     for lfile in possible_local_files:
         if lfile.endswith(".rpm") and os.path.exists(lfile):
             return
+        schemes = dnf.pycomp.urlparse.urlparse(lfile)[0]
+        if schemes and schemes in ('http', 'ftp', 'file', 'https'):
```

Comment:
if schemes in ('http', 'ftp', 'file', 'https'):
should be enough
Also in variable schemes is only one scheme. Rename variable to singular "scheme"?